app/controllers/RevaluationController.scala

```diff
@@ -57,6 +57,8 @@ class RevaluationController @Inject()( authAction: AuthAction,
         logger.debug(s"[RevaluationController][post][POST] : ${request.body}")
         revalForm.bindFromRequest.fold(
           formWithErrors => {
+
```

Comment:
can comments be removed?

Reply:
yup sure 👍

Reply:
I think I removed it in final push, may be you were looking at the commit